pkg/daemon/discover/discover.go

```diff
@@ -35,19 +36,41 @@ import (
 )
 
 var (
-	logger          = capnslog.NewPackageLogger("github.com/rook/rook", "rook-discover")
-	AppName         = "rook-discover"
-	NodeAttr        = "rook.io/node"
-	LocalDiskCMData = "devices"
-	LocalDiskCMName = "local-device-"
+	logger                                  = capnslog.NewPackageLogger("github.com/rook/rook", "rook-discover")
+	AppName                                 = "rook-discover"
+	NodeAttr                                = "rook.io/node"
+	LocalDiskCMData                         = "devices"
+	LocalDiskCMName                         = "local-device-"
+	probeInterval                           = 30 * time.Second
+	nodeName, namespace, lastDevice, cmName string
+	cm                                      *v1.ConfigMap
 )
 
 func Run(context *clusterd.Context) error {
 	if context == nil {
 		return fmt.Errorf("nil context")
 	}
-	nodeName := os.Getenv(k8sutil.NodeNameEnvVar)
-	namespace := os.Getenv(k8sutil.PodNamespaceEnvVar)
+	nodeName = os.Getenv(k8sutil.NodeNameEnvVar)
+	namespace = os.Getenv(k8sutil.PodNamespaceEnvVar)
+	cmName = LocalDiskCMName + nodeName
```

Comment:
do we calculate the cmName in a few different places? maybe a helper function to make sure they don't get out of sync?

Reply:
this is the only place cmName is calculated.